libraries/AP_NavEKF2/AP_NavEKF2_core.cpp

```diff
@@ -42,10 +42,11 @@ NavEKF2_core::NavEKF2_core(void) :
 }
 
 // setup this core backend
-bool NavEKF2_core::setup_core(NavEKF2 *_frontend, uint8_t _imu_index, uint8_t _core_index)
+bool NavEKF2_core::setup_core(NavEKF2 *_frontend, uint8_t _gyro_index, uint8_t _accel_index, uint8_t _core_index)
```

Comment:
i don't think changing setup_core() API is needed? the selections get overwritten anyway